example/src/main/resources/io/netty/example/ocsp/netty_io_chain.pem

```diff
@@ -0,0 +1,63 @@
+-----BEGIN CERTIFICATE-----
```

Comment:
That cert chain is actually not generated. It's the real cert chain for https://netty.io and it was simply pulled with Chrome's export feature.